base/collect_database.go

```diff
@@ -0,0 +1,315 @@
+package base
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/itchyny/gojq"
+	log "github.com/iter8-tools/iter8/base/log"
+
+	"text/template"
+	"time"
+
+	"sigs.k8s.io/yaml"
+)
+
+type CollectDatabaseTemplateInput struct {
```

Comment:
This type doesn't appear to be used anywhere. Should be removed?

Reply:
This is now being used properly in the tests.

Reply:
Could we please rename it as `type collectDatabaseTemplateInput` (note the small c), and also simply move the type definition to the test (since it is intended for use only in the test).